src/modules/core/components/SingleUserPicker/SingleUserPicker.css

```diff
@@ -57,7 +57,7 @@
 .baseInput {
   flex: 1 1 0;
   margin-left: 10px;
-  max-width: 340px;
+  max-width: 305px;
```

Comment:
Why was this reduced ?

If it's just for single instance do it in the spefic component, otherwise, I don't this change is appropriate _(as user addresses tend to be quite long)_

Reply:
I think, if I remember correctly, it was reduced because of this: `Use of inconsistent inputs (modals) - long width, wrong height`

Should I change it back?

Reply:
Nah, leave it.